media-gfx/qimgv/qimgv-0.7.3.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils xdg-utils
+
+DESCRIPTION="A cross-platform image viewer with webm support. Written in qt5"
+HOMEPAGE="https://github.com/easymodo/qimgv"
+SRC_URI="https://github.com/easymodo/qimgv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="kde video"
```

Comment:
USE `kde` is not a good choice here imo, it should describe the feature instead of the dependency - as this does not seem to provide any kind of *KDE* support but uses a tier1 framework (=library) to enable a blur(?) feature.

Reply:
Reading the upstream description it may be limited to Plasma runtime support, but the code does not check for that, so it should work anywhere else too...

Reply:
changed use flag from `kde` to `blur`